### PR TITLE
Adding lots of new resources, a couple new categories

### PR DESCRIPTION
**Pull request by karllhughes**
Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md).
- [x] Your changes are made in the [README](../README.md) file, not the auto-generated JSON. 
- [x] Your additions are ordered alphabetically.
- [x] Your submission has a useful description.
- [x] Each table column should be padded with one space on either side.
- [x] You have searched the repository for any relevant issues or PRs.
- [ ] Any category you are creating has the minimum requirement of 3 items.

### CODE REVIEW

**Comment by karllhughes** [comment]
I have a couple questions based on this PR:

- I added a `Text Analysis` section, but it only has two items. I can move them back into `Development` if you prefer? I just saw the rule about 3 per section.
- The Feedbin API actually uses Basic Auth with a username and password. Since that's not an option, I classified it as OAuth. Is this okay?

Thanks for considering my PR, and great work on this project!

File: README.md [hunk 1]
```diff
@@ -390,7 +418,7 @@ API | Description | Auth | HTTPS | Link |
 | Wordnik | Dictionary Data | No | No | [Go!](http://developer.wordnik.com) |
 | Yelp | Find Local Business | `OAuth` | Yes | [Go!](https://www.yelp.com/developers/documentation/v3) |
 
-### Open Source projects
+### Open Source Projects
```
Comment (karllhughes) [inline README.md]: Hope this is okay. The capitalization inconsistency bothered me a little 😬

**Comment by mikestreety** [comment]
Looks good to me! As it's so big, just thought I'd get a second opinion ;)

File: README.md [hunk 2]
```diff
@@ -494,9 +526,16 @@ API | Description | Auth | HTTPS | Link |
...
 | Pro Motocross | The RESTful AMA Pro Motocross lap times for every racer on the start gate | No | No | [Go!](http://promotocrossapi.com) |
 | Strava | Connect with athletes, activities and more | `OAuth` | Yes | [Go!](https://strava.github.io/api/) |
+| SuredBits | Query sports data, including teams, players, games, scores, and statistics | No | No | [Go!](https://suredbits.com/api/) |
 | UFC Data | Ultimate Fighting Championship information for events and fighters | No | No | [Go!](http://ufc-data-api.ufc.com/) |
 | Wger | Workout manager data as exercises, muscles or equipment | `apiKey` | Yes | [Go!](https://wger.de/en/software/api) |
 
+### Text Analysis
+API | Description | Auth | HTTPS | Link |
+|---|---|---|---|---|
+| Aylien Text Analysis | A collection of information retrieval, and natural language APIs | `apiKey` | Yes | [Go!](http://docs.aylien.com/) |
```
Comment (davemachado) [inline README.md]: `retrieval, and natural` - no comma here

File: README.md [hunk 3]
```diff
@@ -377,6 +404,7 @@ API | Description | Auth | HTTPS | Link |
 | Open Government, India | Indian Government Open Data | `apiKey` | Yes | [Go!](https://data.gov.in/) |
 | Open Government, New Zealand | New Zealand Government Open Data | No | Yes | [Go!](https://www.data.govt.nz/) |
 | Open Government, USA | United States Government Open Data | No | Yes | [Go!](https://www.data.gov/) |
+| Open Skills API | Job titles, skills, and related jobs data | No | No | [Go!](https://github.com/workforce-data-initiative/skills-api/wiki/API-Overview) |
```
Comment (davemachado) [inline README.md]: No need for API in the title

File: README.md [hunk 4]
```diff
@@ -560,6 +599,14 @@ API | Description | Auth | HTTPS | Link |
 |---|---|---|---|---|
 | Universities List | University names, countries and domains | No | Yes | [Go!](https://github.com/Hipo/university-domains-list) |
 
+### URL Shorteners
+API | Description | Auth | HTTPS | Link |
+|---|---|---|---|---|
+| Bitly | URL shortener and link management | `OAuth` | Yes | [Go!](http://dev.bitly.com/get_started.html) |
+| ClickMeter | Monitor, compare and optimize your marketing links | `apiKey` | Yes | [Go!](https://support.clickmeter.com/hc/en-us/categories/201474986) |
```
Comment (davemachado) [inline README.md]: for continuity, should have an oxford comma after `compare`

File: README.md [hunk 5]
```diff
@@ -45,9 +46,11 @@ Please note a passing build status indicates all listed APIs are available since
 * [Shopping](#shopping)
 * [Social](#social)
 * [Sports & Fitness](#sports--fitness)
+* [Text Analysis](#text-analysis)
```
Comment (davemachado) [inline README.md]: Regarding 2 entries in the `Text Analysis` section, I'd prefer if we could find a 3rd service just to keep the rule in standing order, but I understand the hesitation of later having to break these two entries out again if we just lob them into `Development` now. @karllhughes , maybe you could find a third one? 😄

Comment (mikestreety) [reply README.md]: https://www.ibm.com/watson/services/natural-language-understanding/ 
https://cloud.google.com/natural-language/ 

🤔

Comment (davemachado) [reply README.md]: That works 💯

**Comment by karllhughes** [comment]
Thanks for the feedback. Will make some updates and get a revision up soon 😄

**Comment by karllhughes** [comment]
@davemachado I've made updates based on your feedback and @mikestreety's suggestions for text analysis APIs.

I've got more to add later, but just wanted to start somewhere. Let me know if there's anything else I can add.

**Comment by davemachado** [comment]
Thanks @karllhughes !

**Pull request merged**